Client/Frontend/Settings/LoginDetailViewController.swift

```diff
@@ -224,6 +203,17 @@ extension LoginDetailViewController: UITableViewDataSource {
         loginCell.delegate = self
         return loginCell
     }
+    
+    fileprivate func setCellSeparatorHidden(_ cell: UITableViewCell) {
+        // Prevent seperator from showing by pushing it off screen by the width of the cell
+        cell.separatorInset = UIEdgeInsets(top: 0, left: cell.bounds.width, bottom: 0, right: 0)
```

Comment:
See #9239 : This should be `cell?.separatorInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: cell?.bounds.width ?? 0)` if #9239 is accepted

Reply:
@michaelbiv This needs to be updated then before we can merge this? #9239 was merged.

Reply:
Ok, I'll get it updated

Reply:
@michaelbiv any update on this?